_sass/_base.scss

```diff
@@ -15,3 +15,17 @@ $sidenav-width: 250px;
 	-moz-box-sizing: border-box;
 	box-sizing: border-box;
 }
+
+table{
```

Comment:
Why was this change needed? I'm surprised that the default table style doesn't work...

Reply:
Without this change, the borders of the table do not appear on the web page